saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationClassifiers.scala

```diff
@@ -17,22 +17,22 @@ object EntityRelationClassifiers {
   object OrganizationClassifier extends Learnable(tokens) {
     def label: Property[ConllRawToken] = entityType is "Org"
     override lazy val classifier = new SparseNetworkLearner()
-    override def feature = using(word, windowWithin[ConllRawSentence](EntityRelationDataModel, -2, 2, List(pos)), phrase,
+    override def feature = using(word, tokensWithinWindowPos, phrase,
```

Comment:
I am not sure about the answer to your question; I don't see any reason why they can not be separate PRs. 
But it looks to me that the changes are orthogonal. And given that the changes are not extensive, I am fine with the PR as it is. 

As for future, I definitely prefer small and focused PRs.